secboot/secboot_sb.go

```diff
@@ -188,3 +194,67 @@ func UnlockEncryptedVolumeWithRecoveryKey(name, device string) error {
 
 	return nil
 }
+
+func ActivateVolumeWithKey(volumeName, sourceDevicePath string, key []byte, options *ActivateVolumeOptions) error {
```

Comment:
all  these new functions don't seem to be tested

Reply:
Sure, though this one is a wrapper to hide the secboot package. But the other functions need tests.